.github/workflows/nightly_or_release.yml

```diff
@@ -26,13 +26,19 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      # Get the source tag
+      - name: Source tag
+        id: source_tag
+        shell: bash
+        run: echo ::set-output name=SOURCE_TAG::${GITHUB_REF#refs/tags/}
```

Comment:
@mossroy I have copied code from https://github.community/t/how-to-get-just-the-tag-name/16241/11 to output the source tag. The output from this step is then read into line 41 below to set the TAG_NAME. Untested so far.

Reply:
Needs to be tested (including on nightly), but looks good